External/SPMmodified/xASL/xASL_bids_csv2tsvReadWrite.m

```diff
@@ -64,6 +64,15 @@
     end
 end
 
+if strcmp(Fext, '.tsv') && ~exist(PathIn, 'file')
```

Comment:
This was not what this function was intended to. This function was to read a CSV-file and at the same time convert to a TSV-file, which happens a lot when taking output of LST, CAT12, etc, and converting to BIDS (BIDS likes TSV).

Otherwise, we would simply use `xASL_tsvRead` or `xASL_csvRead`.

Was this not clear from the header?

I'm not sure your change here changes the concept, because if we already have a TSV-file, LST outputs a CSV-file with the same name, it should always overwrite. Now, it would take the old TSV-context and save it as if this was the new TSV.

So we should probably revert your edit here, but the header should be more clear. And rename this function, so it is more clear, e.g. `xASL_bids_csv2tsv.m` instead of ReadWrite

Reply:
I don't want to do "a Henk" - but please read my comments before you reply :)
I know what was this issue supposed to do - previously, it saved the CSV in a way that it was conflicting and crashing during parallelization, so we had to save it elsewhere, convert CSV to TSV and then move TSV to the population directory. See all my comments above. I'm fine with the csv2tsv function but it has to be run in the subject dir...

Reply:
We can remove this commit, but I just wanted to catch all possible misuses of this function. So this is a safety net, not a function ;)But we can remove it if you want...